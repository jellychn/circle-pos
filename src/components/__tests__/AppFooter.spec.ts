import { shallowMount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'
import AppFooter from '../AppFooter.vue'

describe('AppFooter.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(AppFooter)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
