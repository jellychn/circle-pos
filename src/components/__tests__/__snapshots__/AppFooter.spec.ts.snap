// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppFooter.vue > matches snapshot 1`] = `
"<div data-v-6704560a="" class="footer">
  <p data-v-6704560a="" class="copyright">Â© 2024 CirclePOS.com All Rights Reserved.</p>
</div>"
`;
